packages/kbn-telemetry-tools/src/tools/serializer.ts

```diff
@@ -157,6 +165,19 @@ export function getDescriptor(node: ts.Node, program: ts.Program): Descriptor |
     return uniqueKinds[0];
   }
 
+  // Support `type MyUsageType = SomethingElse`
```

Comment:
Personally, I'm not a fan of type aliases but see them often enough to warrant adding support here.

Reply:
I agree. The underlying reason to support it is `type MyType = Something & SomethingElse`, which definitely has a point 🙂